tests/rados/test_norecovery_flag_functionality.py

```diff
@@ -97,8 +97,11 @@ def run(ceph_cluster, **kw):
                     f"Scenario2:-The pool pg num is increased to the - {final_pg_num}"
                 )
                 break
+            time.sleep(20)
         old_pg_num = pool_obj.get_pg_autoscaler_value(pool_name, "pg_num_target")
         log.info(f"Scenario2:The current pg number is -{old_pg_num}")
+        utils.configure_osd_flag(ceph_cluster, "set", "norecover")
+        log.info("Scenario2:The norecover is set on the cluster")
```

Comment:
The bulk flag also needs to be removed if we want to make sure that PG count does not decrease when we remove the bulk flag when norecovery flag is set.

Please add step to remove the bulk flag and then continue with verification if the PG count has decreased/increased.

Reply:
I removed the bulk flag.